components/logins/android/build.gradle

```diff
@@ -113,6 +113,7 @@ dependencies {
     implementation 'com.google.protobuf:protobuf-javalite:3.11.4'
     api project(":full-megazord")
     implementation project(":native-support")
+    implementation project(":glean")
```

Comment:
Each component that uses Glean should take a direct dependency on our re-packaged version of it. This prevents apps from trying to use the stand-alone glean in combination with an appservices component (by way of producing an inscrutable build error about duplicate class definitions).